module-presentation/src/main/kotlin/site/yourevents/invitation/facade/InvitationFacade.kt

```diff
@@ -68,6 +69,19 @@ class InvitationFacade(
         )
     }
 
+    fun getInvitationGuests(invitationId: UUID): InvitationGuestResponse {
+        val attend = guestUseCase.getAttendGuestsByInvitation(invitationId)
+            .map { InvitationGuestResponse.GuestResponse.from(it) }
+
+        val notAttend = guestUseCase.getNotAttendGuestsByInvitation(invitationId)
+            .map { InvitationGuestResponse.GuestResponse.from(it) }
```

Comment:
invitationId를 통해 두 종류의 참석자들을 가져올 경우, `GuestService`에서 `Invitation` 객체를 중복되게 두 번 호출해야하는 불편함이 존재할 것 같아 보입니다.

따라서 파사드 클래스에서 `invitation` 객체를 가져오고, 이를 인자로 넘기는 방향은 어떻게 생각하시나요?

또한, 람다식보다는 메서드 참조를 사용하는 편이 좋아보입니다..!
```suggestion
        val attend = guestUseCase.getAttendGuestsByInvitation(invitationId)
            .map(nvitationGuestResponse.GuestResponse::from)

        val notAttend = guestUseCase.getNotAttendGuestsByInvitation(invitationId)
            .map(nvitationGuestResponse.GuestResponse::from)
```

Reply:
그 부분이 훨씬 좋은 것 같습니다. Invitation 객체를 두번 호출해서 불편함이 생기는 부분까지는 생각을 못했는데 좋은 지적 감사합니다. 수정하였습니다!!